src/PointwiseFunctions/GeneralRelativity/WeylElectric.cpp

```diff
@@ -70,11 +104,21 @@ void weyl_electric(
       const tnsr::ii<DTYPE(data), DIM(data), FRAME(data)>& spatial_ricci,   \
       const tnsr::ii<DTYPE(data), DIM(data), FRAME(data)>&                  \
           extrinsic_curvature,                                              \
+      const tnsr::II<DTYPE(data), DIM(data), FRAME(data)>&                  \
+          inverse_spatial_metric) noexcept;                                 \
+  template Scalar<DTYPE(data)> gr::weyl_electric_scalar(                    \
+      const tnsr::ii<DTYPE(data), DIM(data), FRAME(data)>& weyl_electric,   \
+      const tnsr::II<DTYPE(data), DIM(data), FRAME(data)>&                  \
+          inverse_spatial_metric) noexcept;                                 \
+  template void gr::weyl_electric_scalar(                                   \
+      const gsl::not_null<Scalar<DTYPE(data)>*> weyl_electric_scalar_part,  \
+      const tnsr::ii<DTYPE(data), DIM(data), FRAME(data)>& spatial_ricci,   \
```

Comment:
Shouldn’t name this ricci_scalar here...should be weyl_electric...I think clang-tidy failure is complaining about this, using inconsistent parameter names here vs above. Declaration in hpp, definition in cpp (above), and instantiation in cpp (here) should have same names to avoid confusion. A reader of the code might wonder here why you take a tnsr::ii whose name suggests it should be a Scalar.